### PR TITLE
Fix of server google server exception. PublicKeys are still considered…

### PR DESCRIPTION
**Pull request by tunkl**
… valid if communication error occurred.

### CODE REVIEW

**Comment by tunkl** [comment]
I signed it!

**Comment by tunkl** [comment]
Integration test failed because of maven problem. Is there some way how to run it again?

**Comment by ejona86** [comment]
@tunkl, I was able to restart the Travis build and they completed successfully.

File: google-api-client/src/main/java/com/google/api/client/googleapis/auth/oauth2/GooglePublicKeysManager.java [hunk 1]
```diff
@@ -21,11 +21,7 @@
 import com.google.api.client.json.JsonFactory;
 import com.google.api.client.json.JsonParser;
 import com.google.api.client.json.JsonToken;
-import com.google.api.client.util.Beta;
-import com.google.api.client.util.Clock;
-import com.google.api.client.util.Preconditions;
-import com.google.api.client.util.SecurityUtils;
-import com.google.api.client.util.StringUtils;
+import com.google.api.client.util.*;
```
Comment (ejona86) [inline google-api-client/src/main/java/com/google/api/client/googleapis/auth/oauth2/GooglePublicKeysManager.java]: Although checkstyle didn't complain and we are lacking a CONTRIBUTING document to inform you of our style, we [don't use wildcard imports](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports).

**Comment by ejona86** [comment]
I'm a little concerned that swallowing the exception will defeat exponential backoff of callers. But since we don't provide built-in exponential backoff and the API doesn't help callers coordinate (if one is waiting due to backoff, it won't prevent another caller from causing another refresh), I'm thinking this can be fine.

**Pull request closed without merging**